OrbitVulkanLayerClientGgp/LayerLogic.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "LayerLogic.h"
+
+#include <math.h>
+#include <unistd.h>
+
+#include <chrono>
+#include <string>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitCaptureGgpClient/OrbitCaptureGgpClient.h"
+#include "absl/strings/str_format.h"
+
+namespace {
+static constexpr const int kCaptureClientResultSuccess = 1;
+static constexpr uint16_t kGrpcPort = 44767;
+static constexpr double kFrameTimeThresholdMilliseconds = 1000.0 / 60.0;
+static constexpr int64_t kCaptureLengthSeconds = 10;
+}  // namespace
+
+void LayerLogic::StartOrbitCaptureService() {
+  LOG("Starting Orbit capture service");
+  pid_t pid = fork();
+  if (pid < 0) {
+    ERROR("Fork failed; not able to start the capture service");
+  } else if (pid == 0) {
+    std::string game_pid = absl::StrFormat("%d", getppid());
+    // TODO(crisguerrero): Read the arguments from a config file.
+    char* argv[] = {const_cast<char*>("/mnt/developer/OrbitCaptureGgpService"),
+                    const_cast<char*>("-pid"),
+                    game_pid.data(),
+                    const_cast<char*>("-log_directory"),
+                    const_cast<char*>("/var/game/"),
+                    NULL};
+
+    LOG("Making call to %s %s %s %s %s", argv[0], argv[1], argv[2], argv[3], argv[4]);
+    execv(argv[0], argv);
+  }
+}
+
+void LayerLogic::Init() {
+  // Although this method is expected to be called just once, we include a flag to make sure the
+  // gRPC service and client are not initialized more than once.
+  if (data_initialized_ == false) {
+    LOG("Making initialisations required in the layer");
```

Comment:
initialisations --> initializations

Reply:
Oops I totally missed this one. I have updated it (and also another comment I also missed in _OrbitCaptureClientLayer.cpp_. Thanks!